app/src/main/java/com/gophillygo/app/activities/BaseAttractionActivity.java

```diff
@@ -97,6 +100,11 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             Log.d(LOG_LABEL, "Got destination data");
             nearestDestinations = findNearestDestinations();
             locationOrDestinationsChanged();
+
+            // The list and map activities allow updating AttractionFlag of each attraction
```

Comment:
Why is this necessary now when adding to map, but retaining this listener was fine for the other views? The comment sounds like it's necessary for the list as well.

You've removed this or a similar listener previously, and I was able to add it back; it's useful to know when the data changes without managing that state here, which is what `LiveData` is especially good at. Could this be revisited to keep the listener?

Reply:
Retaining the listener actually caused a regression on the list views - on `develop` updating a flag on the Places list view will reset the scroll position of the list.

I agree that it would be good to be able to retain the ability to listen for changes to the data w/o having to manage the state ourselves, and I'm open to suggestions on ways to accomplish that.

Reply:
After testing on `develop`, I haven't been able to reproduce the issue with the scroll list reset. Were there other issues that made removing the listener necessary?

The second data listener that apparently resulted from the map view refactor was unintentional and hopefully can be removed while keeping the added activity abstractions.